[annotables]: https://github.com/stephenturner/annotables/
[annotationhub]: https://bioconductor.org/packages/AnnotationHub/
[appveyor ci]: https://ci.appveyor.com/
[assertive]: https://cran.r-project.org/package=assertive
[basejump]: https://basejump.acidgenomics.com/
[bcbio]: https://bcbio-nextgen.readthedocs.io/
[bcbiobase]: https://bioinformatics.sph.harvard.edu/bcbioBase/
[bcbiornaseq]: https://bioinformatics.sph.harvard.edu/bcbioRNASeq/
[bcbiosinglecell]: https://bioinformatics.sph.harvard.edu/bcbioSingleCell/
[bcl2fastq]: https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html
[biocinstaller]: https://bioconductor.org/packages/BiocInstaller/
[biocmanager]: https://cran.r-project.org/package=BiocManager
[bioconductor]: https://bioconductor.org/
[biostrings]: https://bioconductor.org/packages/Biostrings/
[brightworm]: https://brightworm.acidgenomics.com/
[cell ranger]: https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger
[clusterprofiler]: https://bioconductor.org/packages/clusterProfiler/
[conda]: https://conda.io/
[covr]: https://cran.r-project.org/package=covr
[deseq2]: https://bioconductor.org/packages/DESeq2/
[devtools]: https://cran.r-project.org/package=devtools
[dplyr]: https://dplyr.tidyverse.org/
[edger]: https://bioconductor.org/packages/edgeR/
[eggnog]: http://eggnogdb.embl.de/
[ensembl]: https://www.ensembl.org/
[ensembldb]: https://bioconductor.org/packages/ensembldb/
[entrez]: https://www.ncbi.nlm.nih.gov/search/
[excel]: https://products.office.com/en-us/excel/
[fgsea]: https://bioconductor.org/packages/fgsea/
[genomeinfodb]: https://bioconductor.org/packages/GenomeInfoDb/
[genomeinfodbdata]: https://bioconductor.org/packages/GenomeInfoDbData/
[ggplot2]: https://ggplot2.tidyverse.org/
[github]: https://github.com/
[glue]: https://glue.tidyverse.org/
[go]: http://www.geneontology.org/
[grch37]: https://grch37.ensembl.org/
[gsea]: https://www.gsea-msigdb.org/gsea/
[hgnc]: https://www.genenames.org/
[hms rc]: https://rc.hms.harvard.edu/
[indrops]: https://1cell-bio.com/
[kegg]: https://www.genome.jp/kegg/
[libreoffice]: https://www.libreoffice.org/
[lintr]: https://github.com/jimhester/lintr/
[macos]: https://www.apple.com/macos/
[magrittr]: https://magrittr.tidyverse.org/
[markdown]: https://daringfireball.net/projects/markdown/
[matrix]: https://cran.r-project.org/package=Matrix
[mgi]: http://www.informatics.jax.org/
[msigdb]: http://software.broadinstitute.org/gsea/msigdb/
[nih]: https://www.nih.gov/
[panther]: http://pantherdb.org/
[pathview]: https://bioconductor.org/packages/pathview/
[pheatmap]: https://cran.r-project.org/package=pheatmap
[pointillism]: https://pointillism.acidgenomics.com/
[python]: https://python.org/
[r-wormbase]: https://wormbase.acidgenomics.com/
[r]: https://www.r-project.org/
[rapmap]: https://github.com/COMBINE-lab/RapMap/
[readr]: https://readr.tidyverse.org/
[rio]: https://cran.r-project.org/package=rio
[rlang]: https://rlang.tidyverse.org/
[roxygen2]: https://cran.r-project.org/package=roxygen2
[rstudio]: https://www.rstudio.com/
[salmon]: https://combine-lab.github.io/salmon/
[scater]: https://bioconductor.org/packages/scater/
[scran]: https://bioconductor.org/packages/scran/
[scrna-seq]: https://bcbio-nextgen.readthedocs.io/en/latest/contents/configuration.html#single-cell-rna-sequencing
[seurat]: https://satijalab.org/seurat/
[summarizedexperiment]: https://bioconductor.org/packages/SummarizedExperiment/
[testthat]: https://cran.r-project.org/package=testthat
[tibble]: https://tibble.tidyverse.org/
[tidyr]: https://tidyr.tidyverse.org/
[tidyverse]: https://www.tidyverse.org/
[travis ci]: https://travis-ci.com/
[tximport]: https://bioconductor.org/packages/tximport/
[ucsc]: https://genome.ucsc.edu/
[umap]: https://github.com/lmcinnes/umap/
[umis]: https://github.com/vals/umis/
[windows]: https://www.microsoft.com/en-us/windows/
[wormbase]: https://www.wormbase.org/
[zinbwave]: https://bioconductor.org/packages/zinbwave/
[zinger]: https://github.com/statOmics/zingeR/
